tests/e2e/ResultCacheEndToEndTest.php

```diff
@@ -136,6 +138,15 @@ public function testResultCachePath(): void
 		$this->assertResultCache(__DIR__ . '/resultCache_1.php', sys_get_temp_dir() . '/phpstan/myResultCacheFile.php');
 	}
 
+	public function testSeparateResultCachePath(): void
+	{
+		$this->runPhpstan(0, __DIR__ . '/phpstan_separate_cache.neon');
+
+		$resultCacheFiles = glob(sys_get_temp_dir() . '/phpstan/resultCache-*.php');
```

Comment:
since checking excatly which hashes are generated is a bit too much IMO, I just verified phpstan will create result-cache files including some suffix (aka. the hash), which it did not do before this change.